src/formats/iff.zig

```diff
@@ -115,7 +115,7 @@ pub const ViewportMode = enum(u32) {
     }
 };
 
-pub const Format = enum(u8) {
+pub const IffForm = enum(u8) {
```

Comment:
Why the shortening? `IffFormat` and `format_id` is perfectly fine

Reply:
I'm not shortening, I'm renaming `Format` to `Form` which is the main group chunk of IFF files. I was incorrectly using `Format` when I should have used `Form`.